core/client/fs/src/main/java/alluxio/client/file/AlluxioFileInStream.java

```diff
@@ -162,6 +163,11 @@ public int read() throws IOException {
           handleRetryableException(mBlockInStream, e);
           mBlockInStream = null;
         }
+        if (e instanceof OutOfRangeException) {
+          mContext.acquireMasterClientResource().get()
+              .forceNextSync(new AlluxioURI(mStatus.getPath()));
```

Comment:
After this line, add a log
```
LOG.info("Notified the master that {} should be sync-ed with UFS on the next access", path)
```
I think INFO is needed because we want the user to see and know a retry will just resolve the inconsistency